### PR TITLE
Document how to rename the generated Windows application

### PR DESCRIPTION
**Pull request by domesticmouse**
This PR documents the process for renaming the generated Windows application for a Flutter Windows desktop app.

fixes: https://github.com/flutter/flutter/issues/92376

## Presubmit checklist
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/master/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/desktop.md [hunk 1]
```diff
@@ -425,6 +425,21 @@ Here are some options:
...
+`BINARY_NAME` variable set on line 4 of `windows/CMakeLists.txt` in
+your Flutter project.
+
+```cmake
+cmake_minimum_required(VERSION 3.15)
+project(windows_desktop_app LANGUAGES CXX)
+
+set(BINARY_NAME "YourNewApp")  # Change this line
+
+cmake_policy(SET CMP0063 NEW)
```
Comment (stuartmorgan) [inline src/desktop.md:440]: I would omit this line of context since it's much more likely to become totally wrong than the lines above.

**Review by stuartmorgan — APPROVED** [review]
LGTM as an isolated change, but:
- Shouldn't we have a https://flutter.dev/docs/deployment/windows at this point (in which case this should live there)? We have macOS and Linux, so it seems very strange to not have Windows.
- We should also, either in this change or a follow-up, talk about how to change the version (and maybe description, given the discussion in the issue) in the resource file.

**Review by sfshaza2 — COMMENTED** [review]
I defer to Stuart on this change.

**Comment by domesticmouse** [comment]
I'd suggest we land this change as is and put a task against me to pull out a windows page. I'll get to it after I return from carers leave.

**Comment by stuartmorgan** [comment]
Works for me. I'll file it now.

**Review by sfshaza2 — APPROVED** [review]
lgtm. Thx, @domesticmouse and @stuartmorgan!

**Pull request merged**